app/Insightly/Listeners/SyncContact.php

```diff
@@ -51,7 +51,7 @@ public function handleContactCreated(ContactCreated $contactCreated): void
         );
     }
 
-    private function storeAndLinkContactAtInsightly(Contact $contact): void
+    private function storeAndLinkContactAtInsightly(Contact $contact): int
     {
```

Comment:
You could add a doc comment to annotate that the returned int is a `contactInsightlyId`